lib/login.ts

```diff
@@ -0,0 +1,18 @@
+'use server'
+
+import {redirect} from "next/navigation";
+
+export async function getFormData(formData: FormData) {
+    const login = {
+        email: formData.get('email'),
+        password: formData.get('password'),
+    };
+    console.log(login);
+
+    if(login.email === 'test@test.com' && login.password === 'password'){
+        redirect("/");
+    }
+    else {
+        redirect('http://localhost:3000/login');
```

Comment:
hardcoding is fine for now, but see how you have to write this over and over again. Try to create an environment variable file and put it there so if you need to change it, you will have to change it in only one place

Reply:
Created a .env file for this